fnl/plugins/lsp.fnl

```diff
@@ -0,0 +1,12 @@
+(import-macros {: tx} :magic.macros)
+(local {: autoload} (require :nfnl.module))
+(local lsp (autoload :lspconfig))
+
+(tx :neovim/nvim-lspconfig
+    {:config (fn config []
+               (lsp.clojure_lsp.setup {})
+               (lsp.ts_ls.setup {})
+               (lsp.pylsp.setup {})
+               (lsp.bashls.setup {})
+               (lsp.lua_ls.setup
+                 {:settings {:Lua {:telemetry {:enable false}}}}))})
```

Comment:
I'm thinking about adding mason and mason-lspconfig to ensure these dependencies are installed. Thoughts?

Reply:
Yep I think that's a good pattern! I'm doing that over here + some other nice automatic things:

https://github.com/Olical/dotfiles/blob/d178b62cde51ee28e4cb317ca346208ab706329a/stowed/.config/nvim/fnl/plugins/lsp.fnl